linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
```

Comment:
1. file_name -> file_name_1 : This is needed since the file name in the header isn't always the linted file name. If another file is 'required', that file name will be on the top row, while the 'open/requiring' file will be moved to the bottom. (Found this when testing the require command).
2. line, col -> col_1 : This is to enable use of the near command. Both X:X,X -> single line error and X:X,X:X -> multiline error. The near variable, from what i've seen (Haven't found any documentation on this), is that it is the same as the length of the highlight.

Also removed removed the requirement for special file extensions, since that should be configured in the linter configuration.